addons/survey/static/src/js/survey_form.js

```diff
@@ -523,6 +529,18 @@ publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend(SurveyPreloa
         }
     },
 
+    /**
+     * This method is used to scroll to error generated in the backend.
+     * (Those errors are displayed when the user skip mandatory question(s))
+     */
+    _scrollToFirstError: function() {
+        let $firstNonEmptyError = this.$('.o_survey_question_error :not(:empty):first')
+        if ($firstNonEmptyError.length) {
+            let $scrollToElement = $firstNonEmptyError.parents('.js_question-wrapper')
+            this._scrollToError($scrollToElement);
+        }
```

Comment:
We've decided not to merge new code using jQuery. It should be doable here too.

BTW, I would put it in `ERRORS TOOLS`, near `scrollToError`.